### ISSUE TITLE
[W3C] Summary issue

### ISSUE THREAD

**Issue by alexisben** [noesya/osuny-hugo-theme-AAA#1276]
Vérifier ces nouvelles erreurs du W3C markup validator.

Element “summary” is missing one or more of the following attributes: “aria-checked”, “aria-level”, “role”.

The “aria-expanded” attribute must not be used on any “summary” element that is a summary for its parent “details” element.

**Comment by alexisben** [comment]
https://github.com/validator/validator/issues?q=label:aria-conformance

https://github.com/validator/validator/issues/1930

https://w3c.github.io/html-aria/tests/summary-allowances.html

https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/summary#technical_summary ("No role permitted")
